### PR TITLE
Create CaptureTutorial.md

### PR DESCRIPTION
**Pull request by torbsorb**
This tutorial will be rendered in our Knowledge Base (after the effort to restructure it). See https://zivid.atlassian.net/wiki/spaces/restructKB/pages/268599416/Basic+Capture.

Ideally the code snippets should have been the links. If only github would have rendered them like they do in their comments, like this:

https://github.com/zivid/cpp-extra-samples/blob/54bf513806f72f2e782cc620ccd4e85db96d50b3/source/Camera/Basic/Capture/Capture.cpp#L18-L19

### CODE REVIEW

**Comment by torbsorb** [comment]
This tutorial will be rendered in our Knowledge Base (after the effort to restructure it). See https://zivid.atlassian.net/wiki/spaces/restructKB/pages/268599416/Basic+Capture.

File: source/Camera/Basic/CaptureTutorial.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+## Part 1 - Quick Start
+
+### Connect
+
+First we have to connect to the camera ([go to source][connect-url]).
+```cpp
+auto camera = zivid.connectCamera();
+```
+
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: I would add another line to show how to connect to a specific camera via a serial number.

File: source/Camera/Basic/CaptureTutorial.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+3. More on captures - HDR and 2D
+4. Bonus!
+
+### Prerequisites
+
+You should have installed Zivid SDK and C++ samples. For more details see [Instructions][installation-instructions-url].
+
+## Part 1 - Quick Start
+
+### Connect
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: The crucial thing missing is the application. Without that it is impossible to use this tutorial. Add it as a step before connect or as part of connect step.

File: source/Camera/Basic/CaptureTutorial.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+Now we can capture a frame. The default capture is a single 3D point cloud ([go to source][capture-url]).
+```cpp
+auto frame = camera.capture();
+```
+
+### Save
+
+We can now save our results. By default the 3D point cloud is saved in Zivid format `.zdf` ([go to source][save-url]).
+```cpp
+auto resultFile = "result.zdf";
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: I would like to show that it is possible to save in other file formats as well here. Maybe split into two snippets, first - define your file format, second - save.

File: source/Camera/Basic/CaptureTutorial.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+### Capture
+
+Now we can capture a frame. The default capture is a single 3D point cloud ([go to source][capture-url]).
+```cpp
+auto frame = camera.capture();
+```
+
+### Save
+
+We can now save our results. By default the 3D point cloud is saved in Zivid format `.zdf` ([go to source][save-url]).
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Is it by default? I think you have to specify.

**Comment by SatjaSivcev** [comment]
I like the (go to source) option. We should definitely use this when we have it in samples. For some cases, however, we don't have the code snippet we want to put in tutorial anywhere in the samples. For these cases, I don't think it is a problem. It is more important to have a complete tutorial. For example, showing how to connect to a serial number camera, save ply, etc.

File: source/Camera/Basic/CaptureTutorial.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    std::chrono::milliseconds{ 1200 }, Zivid::CaptureAssistant::AmbientLightFrequency::none);
+
+std::cout << "Running Capture Assistant with parameters: " << suggestSettingsParameters << std::endl;
+const auto settingsVector{ Zivid::CaptureAssistant::suggestSettings(camera, suggestSettingsParameters) };
+```
+
+These settings can be used in an [HDR capture](#capture-hdr), which we will discuss later.
+
+As opposed to manual configuration of settings, there are only two parameters to consider with Capture Assistant.
+1. **Allowed budget for capture** in number of milliseconds.
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: Allowed budget for capture - I think we should be consistent with the terms we use. In Zivid Studio it is Max capture time.

File: source/Camera/Basic/CaptureTutorial.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    1. Minimum allowed budget is 200ms. This allows only one frame to be captured.
+    2. The algorithm will combine multiple frames if the budget allows.
+    3. The algorithm will attempt to cover as much of the dynamic range in the scene as possible.
+    4. A budget of more than 1 second will get good coverage in most scenarios.
+2. **Ambient light compensation**
+    1. May restrict capture assistant to exposure periods that are multiples of the ambient light period.
+    2. 60Hz is found in (amongst others) Japan, Americas, Taiwan, South Korea and Philippines
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: end of sentance - put a dot.

File: source/Camera/Basic/CaptureTutorial.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+    3. 50Hz is found in most rest of the world
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: end of sentance - put a dot.

File: source/Camera/Basic/CaptureTutorial.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+### Manual configuration
+We may choose to configure settings manually ([go to source][settings-url]).
+```cpp
+camera << Zivid::Settings::Iris{ 20 } << Zivid::Settings::ExposureTime{ std::chrono::microseconds{ 8333 } }
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: I think it makes sense to put complete settings and filters here.

File: source/Camera/Basic/CaptureTutorial.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+### Capture 2D
+If we only want to capture a 2D image, which is faster than 3D, we can do so via the 2D API ([go to source][capture2d-url]).
+```cpp
+auto frame = camera.capture2D(settings);
+```
+
+Note that settings are passed as argument. These must be initialized first, and optionally modified ([go to source][settings2d-url]).
+```cpp
+auto settings = Zivid::Settings2D();
+settings.set(Zivid::Settings2D::ExposureTime{ std::chrono::microseconds{ 10000 } });
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: I would add compete settings relevant for 2D capture here.

File: source/Camera/Basic/CaptureTutorial.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+settings.set(Zivid::Settings2D::Gain{ 1.0 });
+settings.set(Zivid::Settings2D::Iris{ 35 });
+```
+
+### Capture HDR
+As was revealed in the [Capture Assistant](#capture-assistant) section, a capture may consist of multiple frames. In order to capture multiple frames, and combine them ([go to source][captureHDR-url])
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: The second sentence doesn't seem to be completed.

File: source/Camera/Basic/CaptureTutorial.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+```cpp
+const auto hdrFrame{ Zivid::HDR::capture(camera, settingsVector) };
+```
+It is possible to manually create the `settingsVector`, if not set via [Capture Assistant](#capture-assistant).
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: I would add code snippet for manually creating the settingsVector

File: source/Camera/Basic/CaptureTutorial.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+```cpp
+Add code here when sample has been updated
+```
+see issue https://github.com/zivid/cpp-extra-samples/issues/70
+
+
+## Bonus
+
+### Connect to a virtual camera [Optional]
```
Comment (SatjaSivcev) [inline source/Camera/Basic/CaptureTutorial.md]: I think this should be above in the 'Capture' step. We want that people who don't have a camera to see this. If it is at the bottom it is not going to be seen straight away.

File: source/Camera/Basic/CaptureTutorial.md (hunk 3 above)
Comment (torbsorb) [reply source/Camera/Basic/CaptureTutorial.md]: I really wanted to keep the first part of the tutorial as clean as possible. However, I realize that the flow would be better if some code was co-located. How about two tutorials? One super-basic, and one quite basic? Maybe CaptureTutorial.md and CaptureTutorial_Complete.md / _Full / _Advanced? (maybe customers would find it advanced?)

Comment (SatjaSivcev) [reply source/Camera/Basic/CaptureTutorial.md]: I am a bit sceptical as I don't think we have much to show to have a good reason to have two (which may be good as it shows out sdk is so simple). However, if you havr ideas, let's give it a go.

Comment (torbsorb) [reply source/Camera/Basic/CaptureTutorial.md]: I'm working on two tutorials now. We can later decide to kill the short one.

File: source/Camera/Basic/CaptureTutorial.md [hunk 13]
```diff
@@ -0,0 +1,216 @@
...
+   2. [Manual Configuration](#manual-configuration)
+      1. [Single](#single-frame)
+      2. [HDR](#hdr-frame)
+      3. [From File](#from-file)
+      4. [2D](#2d-settings)
+4. [Capture](#capture)
+    1. [HDR](#capture-hdr)
+    2. [2D](#capture-2d)
+5. [Save](#save)
+
```
Comment (chrisasc) [inline source/Camera/Basic/CaptureTutorial.md]: Maybe also include how to read a .zdf from file?

**Pull request merged**